.github/workflows/e2e-parallel-full.yml

```diff
@@ -52,11 +51,9 @@ jobs:
             tenant_name: 'nginx'
             deployment_order:
               [
-                'module.e2e_test.module.aws_vpc',
+                'module.e2e_test.module.vpc',
```

Comment:
when I tested this example, it works with just a simple `terraform apply` but leaving this split out to be safe